pylutron/__init__.py

```diff
@@ -779,7 +779,7 @@ def type(self):
   @property
   def is_dimmable(self):
     """Returns a boolean of whether or not the output is dimmable."""
-    return self.type not in ('NON_DIM', 'NON_DIM_INC', 'NON_DIM_ELV', 'EXHAUST_FAN_TYPE', 'RELAY_LIGHTING') and not self.type.startswith('CCO_')
+    return not self.type.startswith('NON_DI') and not self.type.startswith('CCO_') and not self.type.startswith('SWI') and not self.type.startswith('RELAY') and not self.type.startswith('HVAC')
```

Comment:
You seem to have droped EXHAUST_FAN_TUPE from the list.

I'm also not sure how this logic is really any different than what existed with the exception that HVAC was added

Reply:
thank you for your time and review. My bad for missing the EXHAUST_FAN_TYPE indeed.

what I tried to do is to include "NON_DIM_FLUORESCENT", "NON_DIM_NEON_CC" and "SWITCHED_MOTOR" output types in the list to classify these as switches and not dimmers.

How do you feel about just simply adding these to the already existing types list instead?